### ISSUE TITLE
PHPUnit no warning for PHPUnit_Framework_MockObject_Exception

### ISSUE THREAD

**Issue by ovr** [sebastianbergmann/phpunit-mock-objects#317]
Hey!

So one of my co-workers write code with 

```php
class User extends BaseObject {
    protected static function initStorages()
    {
         static::method();
    }
}
```

In this case `static` was unexpected and unneded, because no one will extends from User

It's was rly hard to find the problem
Because no warning messages was occoruped, I debug code and find that
Mock can throw exception that implements PHPUnit_Framework_MockObject_Exception

In phpunit

![image](https://cloud.githubusercontent.com/assets/572096/16894056/11daa78a-4b86-11e6-8063-07c9e9d14235.png)

After it I find class and template in mock build `Generator\mocked_static_method.tpl.dist`
Can you setup message for it like

```twig
    {modifier} function {reference}{method_name}({arguments_decl}){return_delim}{return_type}
    {
        throw new PHPUnit_Framework_MockObject_BadMethodCallException(
            "It's not possible to call {method_name} in MockObject"
        );
    }
```

This will be very helpfull to detect a warning

I would like to send PR for this, but my english is not okey
And I cannot suggest a good message for PHPUnit_Framework_MockObject_BadMethodCallException

Thank you 🐻

**Comment by ovr** [comment]
ping @sebastianbergmann

**Comment by eelf** [comment]
This is a duplicate of https://github.com/sebastianbergmann/phpunit-mock-objects/issues/327 which was released as 3.2.7

**Comment by ovr** [comment]
Seems to be, Thanks @eelf for ping
